services/system/TransactionSys/src/TransactionSys.cpp

```diff
@@ -293,6 +294,9 @@ namespace SystemService
                "transaction references non-existing block");
       }
 
+      Actor<CpuSys>     cpuSys(TransactionSys::service, CpuSys::service);
+      Actor<AccountSys> accountSys(TransactionSys::service, AccountSys::service);
```

Comment:
Do you not like the `to<Service>.action` syntax? I like how it reads

Reply:
`to` doesn't work here, because it uses state set up by the dispatcher.  `processTransaction` is a direct export that doesn't go through the dispatcher.